contracts/kleros/TalentLayerMultipleArbitrableTransaction.sol

```diff
@@ -192,6 +213,9 @@ contract TalentLayerMultipleArbitrableTransaction {
 
         transaction.amount -= _amount;
         _release(transaction.sender, transaction.token, _amount);
+
+        emit Payment(PaymentType.Reimburse, _amount, transaction.token, transaction.jobId);
```

Comment:
This emit makes the emit in _distributeMessage redundant. We could remove the PaymentCompleted event for gas optimization.